ui/lib/config-ui/addon/components/messages/page/list.hbs

```diff
@@ -21,15 +21,15 @@
     {{/if}}
   </:toolbarFilters>
   <:toolbarActions>
-    <ToolbarLink
-      data-test-toolbar-create-message
-      @route="messages.create"
-      @query={{hash authenticated=@authenticated}}
-      @type="add"
-      aria-label="message create"
-    >
-      Create message
-    </ToolbarLink>
+    <Hds::Button
+      @text="Create message"
+      @icon="plus"
+      @color="tertiary"
+      class="toolbar-link"
```

Comment:
Could you update these class to so they are consistent with the other HDS toolbar buttons we have?
```
@color="secondary"
class="toolbar-button"